doc/contributing/collaborator-guide.md

```diff
@@ -532,7 +532,7 @@ The TSC serves as the final arbiter where required.
    [build](https://github.com/nodejs/build/issues) repositories, open new
    issues. Run a new CI any time someone pushes new code to the pull request.
 4. Check that the commit message adheres to [commit message guidelines][].
-5. Add all necessary [metadata](#metadata) to commit messages before landing. If
+5. Add all necessary [metadata](git-node-metadata) to commit messages before landing. If
```

Comment:
It looks like there was some refactoring around metadata on the page, which led to this broken `#metadata` link which led to nowhere. So I've changed it to the relevant